.github/workflows/release.yml

```diff
@@ -27,9 +27,30 @@ jobs:
       - name: Check
         run: twine check dist/*
 
-      - name: Publish
+      - name: Upload packages artifact
         if: github.event_name == 'release'
```

Comment:
I wouldn't skip this — it lets you download the built dists from PRs and normal pushes for inspection. I think, that's useful.
```suggestion
```